boards/stm32l476g-disco/Makefile

```diff
@@ -1,3 +1,6 @@
 MODULE = board
 
+CFLAGS += -DTINYUSB_TUD_RHPORT=1
+CFLAGS += -DTINYUSB_TUH_RHPORT=1
+
```

Comment:
Isn't better to move it to `periph_conf.h` with some `#ifdef MODULE_TINYUSB` instead ? 
We can also default `stm32f429i-disc1` TINYUSB_TUD_RHPORT and TINYUSB_TUH_RHPORT to 1